doc/tutorials/transformer/te_init.ipynb

```diff
@@ -0,0 +1,283 @@
+{
```

Comment:
<p>I think, we need to install this directly into the transformer-extensions when it is used by certain models.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/data-science-sandbox/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>